media/vp9-codec-string.js

```diff
@@ -0,0 +1,25 @@
+if (MediaSource.isTypeSupported('video/webm; codecs="vp9"')) {
+  log('> VP9, Profile 0, level 1 (later fields defaulted)');
```

Comment:
I've always thought of the the old string is completely ambiguous for profile/level. I've not been around long enough to say whether it was originally intended to just imply profile 0, but in practice its used (regrettably) for lots of other profiles/levels.